apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/codegen/kotlin/KotlinCodeGen.kt

```diff
@@ -251,20 +251,17 @@ internal object KotlinCodeGen {
     }
   }
 
-  private fun FieldType.readListItemCode(optional: Boolean): CodeBlock {
+  private fun FieldType.readListItemCode(): CodeBlock {
```

Comment:
We already check null value before this like, so they are never null after this point.